### ISSUE TITLE
"View This Post on Instagram" shows

### ISSUE THREAD

**Issue by watkins656** [sugarshin/react-instagram-embed#240]
The post is not actually showing, just a generic link with "View This Post on Instagram" that takes you to the post on instagram's site.

**Comment by NimishVerma** [comment]
Yes, is there any way to remove that? If you look at the embedded from Instagram, it actually plays video without opening on instagram.
